static/js/invite.js

```diff
@@ -75,9 +75,21 @@ function submit_invitation_form() {
                     invitee_emails_errored.push(value[0]);
                 });
 
+                var non_admin_error = _('Organization administrators can reactivate deactivated users from organization settings.');
+                var admin_error = _('You can reactivate deactivated users from');
+                var check_if_admin = false;
+                var reactivate_link = _('organization settings.');
```

Comment:
Read our "translating" documentation -- `_(` works for Python code, but is totally invalid for JS code.  You want `i18n.t()`.  It also does not work to break up sentences in this way -- other languages have different word orders.

Reply:
I have updated the translation. But, I don't think we can combine the admin_error and reactivation link as I have to convert only 'Organization settings' into a link and the rest must be text. We can combine them both if we can convert the whole sentence into a link.

Reply:
We can have the link HTML present in the sentence in the JS file.